### ISSUE TITLE
The guide attribution is not same as the design - High priority

### ISSUE THREAD

**Issue by gkwan-ibm** [OpenLiberty/openliberty.io#806]
The design is like
![screen shot 2018-11-23 at 2 54 10 pm](https://user-images.githubusercontent.com/29576299/48958854-c7d61e80-ef2f-11e8-8ace-ff9ff5876192.png)

The current implementation is like
![screen shot 2018-11-23 at 2 57 37 pm](https://user-images.githubusercontent.com/29576299/48958942-2a2f1f00-ef30-11e8-9625-4a06d10ce2bf.png)

- [ ] the style look different
- [ ] missing "by Open Liberty" phase

**Comment by justineechen** [comment]
Based off of our discussion this morning the guides-dev team will update the implementation, @steven1046 @kinueng

**Comment by steven1046** [comment]
I believe the "in Open Liberty" should be added to the text in each guide under the `== Great work! You're done!` section.

**Comment by gkwan-ibm** [comment]
I reopen because the fix was partial. Based on the design, the statement should be as following:
- Injecting dependencies into microservices **by Open Liberty** is licensed under CC BY-ND 4.0

The phase **by Open Liberty**  is missing.

To be discussed in future:
if the guide is contributed by an non IBMer, will the statement be as following?
- My guide X by MyName is licensed under CC BY-ND 4.0

**Comment by steven1046** [comment]
https://github.com/OpenLiberty/guides-common/pull/213

**Comment by justineechen** [comment]
Would it be possible to add a variable like {author} ? And we can set the default to Open Liberty? 
That way we can insert author names when we have outside contributors.

**Comment by steven1046** [comment]
Yes we can add a variable, i am trying to get it to default to Open Liberty but adding liquid syntax inside of an included asciidoc isn't working at the moment. I'll keep working to get it.

**Comment by justineechen** [comment]
@steven1046 Thank you!

**Comment by steven1046** [comment]
Could we add `:guide-author: Open Liberty` to all of our guides or is that too much?

**Comment by justineechen** [comment]
Not at all! We can add that to all the guides.

**Comment by steven1046** [comment]
Awesome, i can deliver the changes to use that tag in the attribution.

**Comment by steven1046** [comment]
The pull request attached to this issue will now use that tag.

**Comment by steven1046** [comment]
https://github.com/OpenLiberty/guides-common/pull/213 uses the `:guide-author:` attribute to specify the author.
